packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -220,8 +220,6 @@ export const SUPPORTED_CHAIN_IDS = [
   '0xa86a',
   // Polis Mainnet
   '0x518af',
-  // Aurora Mainnet
-  '0x4e454152',
```

Comment:
The API that this file hits in turn hits CoinGecko, not Infura, so I don't think we want to remove this.